backend/test/integration/openai.test.ts

```diff
@@ -122,16 +118,19 @@ describe('OpenAI Integration Tests', () => {
 		// check the chat history has been updated
 		expect(chatHistory.length).toBe(1);
 		// system role is added to the start of the chat history
-		expect(chatHistory[0].completion?.role).toBe('system');
-		expect(chatHistory[0].completion?.content).toBe(systemRoleDefault);
+		expect('completion' in chatHistory[0]);
```

Comment:
You appear to be missing an assertion here:

`expect( ... ).toBe(true)`

But this feels a little clunky. Can you not do this?

```
if ('completion` in chatHistory[0]) {
  ...
} else {
  throw Error(`unexpected chat message, type=${chatHistory[0].type}`);
}
```

Reply:
no longer relevant, this file no longer exists. Replaced with _unit/openai.test.ts_